### PR TITLE
Get the build workflow working with Acquia

### PR DESCRIPTION
**Pull request by pfrilling**
## Summary
This merges the work from @edaemon's branch in RIG-28/implement-cicd and the composer lock updates from my RIG-28/update-lock-file branches and expands the workflow to run through the workflow and successfully deploy to Acquia.

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | n/a
| Did you perform browser testing? | yes
| Risk level | Mediua
| Relevant links | [RIG-28](https://thinkoomph.jira.com/browse/rig-28)

### CODE REVIEW

**Comment by ben-hamelin** [comment]
Concur!
/approved

**Comment by pfrilling** [comment]
@ben-hamelin, I made a few modifications to this PR to get things working in ACSF. I've confirmed that sites are installed properly now and have the correct permissions for authentication with Active Directory.

File: oomph-actions/vars.yml [hunk 1]
```diff
@@ -10,29 +10,50 @@ run_npm_install: no
...
 # Disable the standard Oomph lando npm install.
 run_lando_npm_install: no
 
 # Use pre-installed Docker EE
 lando_install_prereqs: no
 lando_docker_ce: no
 
+# Append the hooks directory to the deploy files variable.
+drupal_deploy_files:
+  - config
```
Comment (ben-hamelin) [inline oomph-actions/vars.yml:23]: Do we need the config folder if we are managing via the installation profile, features, and then site specific? Wondering if we can drop this and let the sites use the default configuration folder?

**Comment by ben-hamelin** [comment]
Ok LGTM!
/approved

**Pull request merged**